src/docs/product/alerts-notifications/metric-alerts.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Metric Alerts
+sidebar_order: 1
+redirect_from:
+  - /workflow/alerts-notifications/alerts/
+  - /workflow/notifications/alerts/
+  - /product/alerts-notifications/alerts/
+---
+
+Sentry users with admin permissions or higher can create alerts. To confirm or set admin permissions, navigate to **Project Settings > Alerts > New Alert**. To change the frequency of your alerts, configure your [Alert Settings](/product/alerts-notifications/alert-settings/).
```

Comment:
From "To confirm or set..." onwards is incorrect. Admin permissions are in member settings. and as mentioned previously alert frequency is primarily controlled in the alert.